components/admin_console/plugin_management/plugin_management.jsx

```diff
@@ -871,6 +876,48 @@ export default class PluginManagement extends AdminSettings {
                     >
                         {this.renderEnablePluginsSetting()}
 
+                        {/* PluginSettings should be defined same way as other admin settings. */}
```

Comment:
nit: This feels like the same comment, but just without the word `TODO`. Is it valuable?